lib/src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ForwardRefRenderFunction } from "react";
+import { textAreaStyle } from "./text-area.css";
+import { TextAreaProps } from "./text-area.types";
+
+const TextAreaComponent: ForwardRefRenderFunction<HTMLTextAreaElement, TextAreaProps> = ({ 
+    id,
+    name,
+    autoComplete = 'off',
+    autoCorrect = 'off',
+    autoFocus,
+    cols = 35,
+    rows = 5,
+    disabled,
+    form,
+    maxLength,
+    minLength,
+    placeHolder = 'Placeholder',
+    readOnly,
+    required,
+    spellCheck,
+    wrap = 'soft',
+    value,
+    className,
+    children,
+    style
+ }, ref) => {
+
+  return (
+    <textarea className={`${textAreaStyle} ${className}`} style={style} ref={ref}
+        autoComplete={autoComplete}
+        autoCorrect={autoCorrect}
+        autoFocus={autoFocus}
+        cols={cols}
+        disabled={disabled}
+        form={form}
+        id={id}
+        maxLength={maxLength}
+        minLength={minLength}
+        name={name}
+        placeholder={placeHolder}
+        readOnly={readOnly}
+        required={required}
+        rows={rows}
+        spellCheck={spellCheck}
+        value={value}
+        wrap={wrap}
```

Comment:
Hey, you don't really have to specify these as the `extends` manages this. Helps to differ from native props of the textarea and the props we are giving by ourselves.

just spread those as `nativeTextAreaProps`